backend/de.metas.monitoring/src/main/java/de/metas/monitoring/adapter/PerformanceMonitoringData.java

```diff
@@ -27,14 +20,20 @@
  * #L%
  */
 
-@UtilityClass
-public class ApiAPMHelper
+package de.metas.monitoring.adapter;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+
+@Getter
+@Setter
```

Comment:
if u are adding the methods i've advised, 
it might me that u would need some `Getter/Setter` annotation on couple of fields only,
most of the fields won't need it.

PS: anyways in case u need getter and setter annotation of class level, it might be that you want to use `@Data` annotation.